### ISSUE TITLE
Creating duplicates of selected nodes and rename them "CURVE"

### ISSUE THREAD

**Issue by luca-amorosi** [krathjen/studiolibrary#433]
Hi, 

I have encountered an issue while trying to save an animation.
I am using `Maya 2022.3` with python 3 and `StudioLibrary 2.9.11`.
If I try to save an animation of a joint which has nurbsCurve shape. StudioLibrary seems to duplicate it and renames it "CURVE".

Below you can find a small code to create a joint an add a nurbsCurveShape to this joint. It also put 2 keyframes in translate.

```
from maya import cmds
jnt = cmds.createNode("joint", name="joint_ctrl")
nurbs_curve_shape = cmds.createNode("nurbsCurve", name="joint_ctrlShape", parent=jnt)
cmds.setKeyframe(f"{jnt}.tx", value=0, time=[0, 0])
cmds.setKeyframe(f"{jnt}.tx", value=10, time=[10, 10])
```

After exporting an animation with the "joint_ctrl" selected, a "CURVE" joint should appear.
I don't know why and this cause errors in rigs since piece of the rig are then randomly duplicated.
![image](https://github.com/krathjen/studiolibrary/assets/69077031/ca65e458-ab0e-46fa-8b9a-44a9a4c7eaac)

**Comment by krathjen** [comment]
Hi @luca-amorosi,

A similar issue was addressed in a recent release. Try downloading the latest version from the website or the main branch. If the issue persists, please reopen the ticket or contact [support@studiolibrary.com](mailto:support@studiolibrary.com)

https://www.studiolibrary.com

Kurt

**Comment by luca-amorosi** [comment]
Hi Kurt, I hope you are doing well !

Thank you for the fast reply. I really appreciate that.

Yes I found out that it was fixed in the latest version, 2.13.1, but it
does not appear in the repository's releases. Does it mean this version is
not production ready yet ?

**Comment by luca-amorosi** [comment]
Okep that is great to know thank you Kurt, and have a nice day ! :)
